uw_sws/models.py

```diff
@@ -19,6 +20,12 @@
     "{year}-{quarter}-{curr_abbr}-{course_num}-{section_id}-{inst_regid}")
 
 
+def date_to_json(dt):
```

Comment:
This needs to be moved to uw_sws.util and renamed to date_to_str().

What was wrong with str(dt)?

Reply:
when the dt is None, str(None) returns 'None'. 
Does it matter?